src/arch/linux/mapping/mapping.c

```diff
@@ -430,7 +453,10 @@ void *mmap_mapping(int cap, void *target, size_t mapsize, int protect)
       (cap & (MAPPING_DPMI|MAPPING_VGAEMU|MAPPING_INIT_LOWRAM|MAPPING_KVM)))
     flags = _MAP_32BIT;
 #endif
-  addr = mmap(target, mapsize, protect,
+  if ((cap & (MAPPING_INIT_LOWRAM|MAPPING_KVM)) && flags != MAP_FIXED)
+    addr = mmap_aligned(target, mapsize, protect);
```

Comment:
Poor name, it made me wtf and
do `man mmap_aligned`. :)
`do_mmap_aligned` or alike, please.